src/envoy/extensions/wasm/wasm.cc

```diff
@@ -0,0 +1,137 @@
+#include "extensions/common/wasm/wasm.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Wasm {
+namespace Istio {
+
+class IstioContext : public Context {
```

Comment:
Can you extract IstioContext to a separated `context.h` file? Extended ABI implemented in other files will need to include and reference this context.

Reply:
Done.